src/ChatWindow.vue

```diff
@@ -43,9 +44,21 @@ export default {
       type: Boolean,
       default: false
     },
+    /* Either agentProfile or agentProfiles is required */
     agentProfile: {
-      type: Object,
-      required: true
+      type: Object
+    },
+    agentProfiles: {
```

Comment:
I'd also change the name from `agentProfiles` to `participants`, what do you think about this?